backend/core/src/listings/listings.service.ts

```diff
@@ -36,12 +40,61 @@ export class ListingsService {
       query.andWhere("property.neighborhood = :neighborhood", { neighborhood: params.neighborhood })
     }
 
-    let listings = await query.getMany()
+    let currentPage: number = params.page
+    let itemsPerPage: number = params.limit
```

Comment:
What happens if there's no page or limit param? Do they default to a value or does this just fail?

Reply:
The default values are `page: 1` and `limit: 10` (https://github.com/CityOfDetroit/bloom/blob/ec9124b6281953e2c5b1ac63a83d38e5ceb3cad5/backend/core/src/shared/dto/pagination.dto.ts#L44). It's possible to set them to something else in the request, but if they are invalid values (e.g. if either is <= 0) we just return all listings.